tests/platform/cloud/OAuth2AcceptanceDataStore.js

```diff
@@ -68,6 +76,30 @@ fluid.defaults("gpii.oauth2.dataStore.acceptanceData", {
                 selectedPreferences: { "": true },
                 revoked: false
             }
+        ],
+        credentialClientTokensIdSeq: 4,
+        credentialClientTokens: [
+            {
+                id: 1,
+                clientId: 5,
+                accessToken: "first_discovery_access_token",
+                revoked: false,
+                allowAddPrefs: true
+            },
```

Comment:
Lets' remove this pre-configured token to verify that granting of a token adds a new entry properly.